samples/net/sockets/echo/boards/cc3220sf_launchxl.conf

```diff
@@ -17,7 +17,7 @@ CONFIG_NET_CONFIG_PEER_IPV4_ADDR=""
 
 # Debugging
 CONFIG_NET_LOG=y
-CONFIG_SYS_LOG_WIFI_LEVEL=4
+CONFIG_WIFI_LOG_LEVEL_DBG=y
```

Comment:
Note:  this already part of PR #10392.

Reply:
ok, someone will need to rebase I guess :)